CircleCI/resources/logs/job-complete.json

```diff
@@ -0,0 +1,62 @@
+{
+  "happened_at": "2021-08-04T19:22:46.224Z",
+  "pipeline": {
+    "id": "xxxxxxxxxxxxxxxxxxxxx",
```

Comment:
would be good to have more demo data with failed, success, job-completed, workflow-completed, failed and success workflow events.
Also would be good to replace <pipeline-number> with some demo ids or numbers, so that it resembles real life data.